web-bluetooth/automatic-reconnect-async-await.js

```diff
@@ -3,9 +3,9 @@ var bluetoothDevice;
 async function onButtonClick() {
   bluetoothDevice = null;
   try {
-    log('Requesting any Bluetooth Device...');
+    log('Requesting any Bluetooth Device... (NOT RECOMMENDED - USE FILTERS)');
```

Comment:
I don't think it's that clear what exacted is not recommended?

```js
navigator.bluetooth.requestDevice({
  acceptAllDevices: true /* NOT RECOMMENDED - USE FILTERS */});
```

Also fwiw: 10% of the devices requests have been using anyNamedDevice().